api/enki.py

```diff
@@ -481,9 +496,16 @@ def __init__(self, _db, collection=None, *args, **kwargs):
                 _db, name="Test Enki Collection", protocol=EnkiAPI.ENKI
             )
             collection.protocol=EnkiAPI.ENKI
-            collection.external_integration.setting(self.ENKI_LIBRARY_ID_KEY).value = u'c'
         if collection not in library.collections:
             library.collections.append(collection)
+
+        # Set the "Enki library ID" variable between the default library
+        # and this Enki collection.
+        ConfigurationSetting.for_library_and_externalintegration(
```

Comment:
This is also the default value that'll appear in the admin interface?

Reply:
No, this is only used in unit tests. When you add a library to an Enki collection in the admin interface, no default value will be given for this configuration setting.